.k8s/.dockerignore

```diff
@@ -0,0 +1,40 @@
+*.pyc
```

Comment:
This file is required so we avoid copying unnecessary files to the docker images with the workspace. This file is a super-set of the .gitignore file